recipes/pyopengl/post-link.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Test for OpenGL Driver on Linux
+if [ `uname` == Linux ]; then
+    $PYTHON -c "import OpenGL.GL"
```

Comment:
I think that means that `$PYTHON` is undefined. The question is now whether this will give us the right `python` if we just use `python`. Would you be up for trying it and seeing how it goes?